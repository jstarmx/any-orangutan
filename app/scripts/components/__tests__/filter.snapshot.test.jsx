import React from 'react';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import renderer from 'react-test-renderer';

import Filter from '../filter';

const store = configureMockStore()({ filter: { activeFilter: 'all' } });

it('renders correctly', () => {
  const tree = renderer.create(
    <Provider store={ store }>
      <Filter activeFilter="all" filter={ () => {} } />
    </Provider>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
