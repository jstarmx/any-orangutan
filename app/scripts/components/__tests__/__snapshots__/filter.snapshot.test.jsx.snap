// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <section
    className="filter"
  >
    filter by:
    <button
      className="filter__button filter__button--active"
      onClick={[Function]}
      type="button"
    >
      all
    </button>
    <button
      className="filter__button filter__button--highlight"
      onClick={[Function]}
      type="button"
    >
      favourites
    </button>
  </section>
</div>
`;
